Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/sheets/person/ContactDiaryPersonBottomSheetDialogViewModel.kt

```diff
@@ -75,12 +86,10 @@ class ContactDiaryPersonBottomSheetDialogViewModel @AssistedInject constructor(
         shouldClose.postValue(null)
     }
 
-    companion object {
-        private const val MAX_PERSON_NAME_LENGTH = 250
-    }
-
     @AssistedInject.Factory
     interface Factory : CWAViewModelFactory<ContactDiaryPersonBottomSheetDialogViewModel> {
         fun create(addedAt: String?): ContactDiaryPersonBottomSheetDialogViewModel
     }
 }
+
+private const val MAX_PERSON_NAME_LENGTH = 250
```

Comment:
Global constant? Why?

Reply:
It's private, not global.

Reply:
In general, companion objects should be avoided.